operator/v1/types_dns.go

```diff
@@ -48,6 +48,11 @@ type DNSSpec struct {
 	// +optional
 	Servers []Server `json:"servers,omitempty"`
 
+	// upstreamConfiguration defines a schema for configuring CoreDNS
+	// to proxy DNS messages to upstream resolvers for the case of the
+	// default (".") server
+	UpstreamConfiguration ForwardPlugin `json:"upstreamConfiguration"`
```

Comment:
The _configuration_ is somewhat redundant because all of this is configuration. IMHO a better name for this field would be `UpstreamResolvers`

Reply:
Great suggestion. I'll go with that field name :+1: